hooks/addon

```diff
@@ -196,6 +200,77 @@ upload-stemcells|us)
   upload_stemcells "$@"
   ;;
 
+runtime-config|rc)
+  describe "" \
+    "The \`netop' user is a local administrator account configured with" \
+    "a 4096-bit RSA SSH key for authentication.  It can be used to perform" \
+    "out-of-band, remote management of BOSH VMs when BOSH is misbehaving." \
+    ""
+  prompt_for do_netop boolean \
+    "Enable the netop account? " -i
+
+  describe "" \
+    "The \`sysop' user is a local administrator account configured with" \
+    "a randomized password, for console-based authentication.  This can be" \
+    "handy in vSphere environments when network-based authentication breaks." \
+    ""
+  prompt_for do_sysop boolean \
+    "Enable the sysop account? " -i
+
+  BOSH_ENVIRONMENT=$BOSH_URL \
+  BOSH_CA_CERT=$(safe read $vault/ssl/ca:certificate) \
+  BOSH_CLIENT="admin" \
+  BOSH_CLIENT_SECRET="$(safe read ${vault}/users/admin:password)" \
+  bosh runtime-config > rc.yml
+
+  if [[ ! -s rc.yml ]]; then
+    echo "--- {}" > rc.yml
+  fi
+
+  if [[ "${do_netop}" == "true" || ${do_sysop} == "true" ]]; then
+    BOSH_ENVIRONMENT=$BOSH_URL \
+    BOSH_CA_CERT=$(safe read $vault/ssl/ca:certificate) \
+    BOSH_CLIENT="admin" \
+    BOSH_CLIENT_SECRET="$(safe read ${vault}/users/admin:password)" \
+    bosh update-runtime-config <( (
+      echo 'releases:'
+      echo '  - name:    os-conf'
+      echo '    version: 20'
+      echo '    url:     https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=20'
+      echo '    sha1:    42b1295896c1fbcd36b55bfdedfe86782b2c9fba'
+      echo
+
+      echo 'addons:'
+      echo '  - name: local-users'
+      echo '    jobs:'
+      echo '      - (( replace ))'
+      echo '      - name:    user_add'
+      echo '        release: os-conf'
+      echo '        properties:'
+      echo '          persistent_homes: true'
+      echo '          users:'
+      if [[ "${do_netop}" == "true" ]]; then
+        echo '            - name: netop'
+        echo '              public_key: (( vault $VAULT_PREFIX "/op/net:public" ))'
+      fi
+      if [[ "${do_sysop}" == "true" ]]; then
+        echo '            - name: sysop'
+        echo '              crypted_password: (( vault $VAULT_PREFIX "/op/sys:password-crypt-sha512" ))'
+      fi
+    ) | VAULT_PREFIX="$vault" spruce merge rc.yml - )
+
+  else
+    BOSH_ENVIRONMENT=$BOSH_URL \
+    BOSH_CA_CERT=$(safe read $vault/ssl/ca:certificate) \
+    BOSH_CLIENT="admin" \
+    BOSH_CLIENT_SECRET="$(safe read ${vault}/users/admin:password)" \
+    bosh update-runtime-config <(
```

Comment:
It would be prudent to explicitly remove the `local-users` add-on and the `os-conf` release in case the runtime config has other addons and releases managed manually.